Nos/Models/Author+CoreDataClass.swift

```diff
@@ -184,14 +183,20 @@ public class Author: NosManagedObject {
             "$reference.marker = 'root' OR $reference.marker = 'reply' OR $reference.marker = nil" +
         ").@count = 0"
         let onlyAuthorsWithStoriesClause = "SUBQUERY(events, $event, \(onlyPostsClause) " +
+            "AND \(onlyUnreadStoriesClause) " +
             "AND \(onlyRecentStoriesClause) " +
             "AND \(onlyRootPostsClause)).@count > 0"
-
-        fetchRequest.predicate = NSPredicate(
+        return NSPredicate(
             format: "\(onlyFollowedAuthorsClause) AND \(onlyAuthorsWithStoriesClause)",
             self,
             since as CVarArg
         )
+    }
+
+    @nonobjc func followedWithNewNotes(since: Date) -> NSFetchRequest<Author> {
+        let fetchRequest = NSFetchRequest<Author>(entityName: "Author")
+        fetchRequest.sortDescriptors = [NSSortDescriptor(keyPath: \Author.hexadecimalPublicKey, ascending: false)]
+        fetchRequest.predicate = followedWithNewNotesPredicate(since: since)
         fetchRequest.fetchLimit = 50
```

Comment:
Maybe we should get rid of this? It should be bounded by the number of people you follow, and I could see some users following hundreds of people and they should be able to see all their followees' stories.
```suggestion

```

Reply:
👌